content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,214 @@
+========
+Chatbots
+========
+
+A *chatbot* is a program designed to mimic a conversation with a live human. Chatbots are assigned
+a script of pre-written steps to follow. These scripts are designed to anticipate a visitor's
+potential response, and lead them through a series of questions and answers the same way an live
+team member would. Chatbots can be customized to fill various roles, depending on the page of the
+website they are assigned and the messages included in the script, among other criteria.
+
+.. image:: chatbot/chatbot-visitor-view.png
+   :align: center
+   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+Build a chatbot
+===============
+
+Before creating a new chatbot, the :guilabel:`Live Chat` application must first be installed on the
+database. This can be done directly from the :menuselection:`Apps` menu by searching for
+:guilabel:`Live Chat` in the :guilabel:`Search bar` and clicking :guilabel:`Install`.
+
+Alternatively, :guilabel:`Live Chat` can be installed and enabled by navigating to the
+:menuselection:`Website application --> Configuration --> Settings` and checking the box labeled
+:guilabel:`Livechat`.
+
+Once the *Live Chat* application has been installed on the database, open it and go to
+:menuselection:`Configuration --> Chatbots`.
+
+.. note::
+    When the *Live Chat* app is installed, a sample chatbot is created, named *Welcome Bot*. As the
```

Comment:
Is *Welcome Bot* something they can delete? Is it just for internal training/instructional use?